scripts/actions/mobskills/gouging_branch.lua

```diff
@@ -0,0 +1,30 @@
+-----------------------------------
+--  Gouging Branch
+--  Description: Drills a single target with a branch. Additional effect: Blind
+--  Type: Physical
+--  Utsusemi/Blink absorb: 2-3 shadows
+--  Range: Melee
+--  Notes: Only used by Stcemqestcint
+-----------------------------------
+require('scripts/globals/mobskills')
```

Comment:
I dont think this are required... Now i'm doubting but i dont think so

Reply:
It’s not, I copied it from the script I already did, removed them.